lib/decidim_app/k8s/organization_exporter.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+module DecidimApp
+  module K8s
+    class OrganizationExporter
+      FORBIDDEN_ENVIRONMENT_KEYS = %w(BACKUP_ENABLED
+                                      BACKUP_S3SYNC_ENABLED
+                                      BACKUP_S3SYNC_ACCESS_KEY
+                                      BACKUP_S3SYNC_SECRET_KEY
+                                      BACKUP_S3SYNC_BUCKET
+                                      BACKUP_S3RETENTION_ENABLED
+                                      DEFAULT_LOCALE
+                                      AVAILABLE_LOCALES
+                                      FORCE_SSL
+                                      SCALEWAY_ID
+                                      SCALEWAY_TOKEN
+                                      SCALEWAY_BUCKET_NAME
+                                      SECRET_KEY_BASE).freeze
+      ORGANIZATION_COLUMNS = %w(id
+                                default_locale
+                                available_locales
+                                users_registration_mode
+                                force_users_to_authenticate_before_access_organization
+                                available_authorizations
+                                file_upload_settings).freeze
+
+      def initialize(organization, logger, export_path, image = "")
+        @organization = organization
+        @logger = logger
+        @export_path = export_path
+        @image = image
+        @database_name = Rails.configuration.database_configuration[Rails.env]["database"]
+      end
+
+      def self.export!(organization, logger, export_path, image)
+        new(organization, logger, export_path, image).export!
+      end
+
+      def self.dumping_database(organization, logger, export_path)
+        new(organization, logger, export_path).dumping_database
+      end
+
+      def export!
+        creating_directories
+        exporting_env_vars
+        exporting_configuration
+      end
+
+      def dumping_database
+        @logger.info("dumping database #{@database_name} to #{organization_export_path}/postgres/#{resource_name}--de.dump")
```

Comment:
```suggestion
        @logger.info("dumping database #{@database_name} to #{organization_export_path}/postgres/#{resource_name}--#{@organization.host}--de.dump")
```

Reply:
Also I don't understand why the exporter should create a filename with the organization host because it dumps the whole database and all Organizations within

Reply:
This change request is still active but I added commit after about other things